### ISSUE TITLE
Firefox | IE compatibility

### ISSUE THREAD

**Issue by jesung1221** [Windrill/SMICTrades#1]
Instead of a quote saying "firefox..." in the main page, 
I suggest we use a little bit of javascript.
So when a user enters the webpage,

<script> alert ("Using firefox and IE 8 or below may cause formatting issues.") </script>

**Comment by Windrill** [comment]
Good idea. Are there any issues on display when you open the website with firefox, before I edit the quote and change the setup/colors?
